### DISCUSSION TITLE
Red exclamation mark on Dev tools icon?

### DISCUSSION THREAD

**Discussion by chinook77** [darkreader/darkreader#9664] [Q&A]
Hi, what does it mean this exclamation mark?

![Screenshot 2022-08-24 100014](https://user-images.githubusercontent.com/19517348/186424791-bbb21327-5fea-48cb-aa42-6deffa1b3553.png)

**Comment by oeloo** [comment]
I have the same question, what is this exclamation mark on 'Dev tools' ?
![image](https://user-images.githubusercontent.com/98819304/190354195-e3f5f876-a378-4bae-8a4f-6339e5ce32fe.png)

Even after clicking the Reset button, this exclamation mark is still present:
![image](https://user-images.githubusercontent.com/98819304/190354591-ada1d70f-39a5-409b-96bc-3ddd5a96f172.png)
Even after rebooting Windows and restarting Chrome browser, this exclamation mark is still present. I use the latest version of Windows , Chrome and Dark Reader extension.

**Comment by Gusted** [comment]
It will be removed in the next version, it doesn't add any value currently. https://github.com/darkreader/darkreader/pull/9807
